src/Components/Button/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import { useState } from "react";
+import { Modal2 } from "../Modal2/Modal2";
+import "../Button/Button.css";
+export const Button = () => {
+  const [show, setShow] = useState(false);
+  return (
+    <>
+      <div className="button-container">
+        <button onClick={() => (!show ? setShow(true) : setShow(false))}>
```

Comment:
We don not need a button to show modal here even if we make a web app.
Hence, modal can be displayed or made visible directly on main app either it is web app or web extension.